Deprecation of Classic Visualization – By November 22, 2024 (#4740)
SumoLogic/sumologic-documentation: blog-service/2024-11-08-search.md

---

> title: Deprecation of Classic Visualization – By November 22, 2024 (Search)

Reviewer: ```suggestion
title: Deprecation of Classic Visualization (Search)
```

image: https://help.sumologic.com/img/sumo-square.png
keywords:
  - classic-visualization
  - search
hide_table_of_contents: true  
---

import useBaseUrl from '@docusaurus/useBaseUrl';

<a href="https://help.sumologic.com/release-notes-service/rss.xml"><img src={useBaseUrl('img/release-notes/rss-orange2.png')} alt="icon" width="50"/></a>

> In May 2024, we introduced new visualization charts for all the Log Search queries, with which you can customize each chart by modifying the available settings and obtain a unified experience across the Sumo Logic platforms.

Reviewer 2: ```suggestion
In May 2024, we introduced new visualization charts for all the Log Search queries, providing enhanced customization options and a consistent experience across the Sumo Logic platform.
```

> **On November 22. 2024**, Sumo Logic will remove the **Switch to Classic Visualization** button, and you will no longer be able to switch to view the charts in classic visualization. By default, all your charts will appear in the New Visualization style.

Reviewer 2: ```suggestion
**On November 22. 2024**, Sumo Logic will remove the **Switch to Classic Visualization** button, and you will no longer be able to switch to view the charts in classic visualization. By default, all your charts will appear in the New Visualization style.

**Effective November 22, 2024**, the **Switch to Classic Visualization** button will be removed, and the classic visualization view will no longer be available. All charts will automatically display in the New Visualization style by default.
```
